app/assets/stylesheets/home.scss

```diff
@@ -8,7 +8,7 @@
 }
 
 .example-chart {
-  background-color: #f3f3f3;
+  background-color: #ffd298;
```

Comment:
Color literals like `#ffd298` should only be used in variable declarations; they should be referred to via variable everywhere else.